src/pages/Lending/hooks/useGetEstimatedFeesQuery.ts

```diff
@@ -13,7 +13,7 @@ export const queryFn = async ({ queryKey }: { queryKey: EstimatedFeesQueryKey })
   const { estimateFeesInput, asset, assetMarketData } = queryKey[1]
 
   // These should not be undefined when used with react-query, but may be when used outside of it since there's no "enabled" option
-  if (!asset || !estimateFeesInput?.to || !estimateFeesInput.accountId) return
+  if (!asset || !estimateFeesInput?.to || !estimateFeesInput.accountId?.length) return
```

Comment:
assuming this was leftover cruft before `| null` updates. changed back in my cleanup pr, so let me know if this was intended for some reason

Reply:
Absolutely a leftover, nice catch!

Reply:
Resolving since this has been tackled in the cleanup PR